taxonomies/categories.txt

```diff
@@ -73257,6 +73257,10 @@ ciqual_food_code:en:6002
 ciqual_food_name:en:Beef, shoulder, raw
 ciqual_food_name:fr:Boeuf, épaule, crue
 
+<en:Beef meat
+en:Beef liver
+fr:Foie de boeuf
```

Comment:
category names should be in plural (it's an issue for many CIQUAL categories that were added as singular); the singular should be added as a synonym